### PR TITLE
feat(tutorial): complete step 2-boeblingen

### PR DESCRIPTION
**Pull request by DanielaImmel**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/LandingPage/_landing-page.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.landing-page__subheading {
+  @include carbon--type-style('productive-heading-03');
+  @include carbon--font-weight('semibold');
+}
+
+.landing-page__p {
+  @include carbon--type-style('productive-heading-03');
+  margin-top: $spacing-06;
+  margin-bottom: $spacing-08;
+}
```
Comment (kellychurchill) [inline src/content/LandingPage/_landing-page.scss]: Missing class `landing-page__r3`

Comment (kellychurchill) [inline src/content/LandingPage/_landing-page.scss]: Missing class `landing-page__label`

Comment (DanielaImmel) [reply src/content/LandingPage/_landing-page.scss]: changes added

File: src/app.scss [hunk 2]
```diff
@@ -1,3 +1,8 @@
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-family.scss';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/breakpoint.scss';
+@import 'carbon-components/scss/globals/scss/typography.scss';
+@import 'carbon-components/scss/globals/scss/vars.scss';
+
 @import 'carbon-components/scss/globals/scss/typography.scss';
```
Comment (ZrianinaMariia) [inline src/app.scss]: Duplicated import.

File: src/app.scss [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
 @import 'carbon-components/scss/globals/scss/vars.scss';
```
Comment (ZrianinaMariia) [inline src/app.scss]: Duplicated import.

**Comment by ZrianinaMariia** [comment]
@DanielaImmel You made it 👍 Looks great 🏆! I've recorded your completion of step 2.

**Pull request closed without merging**